gr-digital/lib/map_bb_impl.h

```diff
@@ -32,8 +32,9 @@ namespace digital {
 class map_bb_impl : public map_bb
 {
 private:
-    unsigned char d_map[0x100];
-    gr::thread::mutex d_mutex;
+    static constexpr size_t s_map_size = 0x100;
+    unsigned char d_map[s_map_size];
+    mutable gr::thread::mutex d_mutex;
```

Comment:
uh, good catch. I'm usually a bit careful when it comes to `mutable`, because especially applied to mutexes that's one great way to hide the fact that something actually might result in a deadlock, but looks appropriate here.